MobiusCore/Source/MobiusLoop.swift

```diff
@@ -104,7 +104,7 @@ public final class MobiusLoop<Model, Event, Effect>: Disposable, CustomDebugStri
 
     // swiftlint:disable:next function_parameter_count
     static func createLoop<C: Connectable>(
-        update: @escaping Update<Model, Event, Effect>,
+        update: Update<Model, Event, Effect>,
```

Comment:
Can we add the follow overload for this:
```swift
static func createLoop<C: Connectable>(
    update: @escaping (Model, Event) -> Next<Model, Effect>,
    ...
)
```

That way most of the existing code should still work as far as I can tell. The only cases which would break would be the onces which explicitly referenced the `Update` typealias.

Reply:
Agreed

Reply:
This `createLoop` appears to be internal. Should I add the overload to `Mobius.loop`?

Reply:
That makes a lot more sense. :-)

Reply:
Done!